datastore/admin-api/DatastoreAdmin.Samples/ImportEntities.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START datastore_admin_entities_import]
+
+using Google.Cloud.Datastore.Admin.V1;
+using Google.LongRunning;
+using Google.Protobuf.WellKnownTypes;
+using System;
+using System.Collections.Generic;
+
+public class ImportEntitiesSample
+{
+    public bool ImportEntities(
+        string projectId = "your-project-id",
+        string inputUrl = "gs://your-project-id-node-perf-test/2020-09-30T08:35:52_77113/2020-09-30T08:35:52_77113.overall_export_metadata",
```

Comment:
This should be a more meaningful value.

Reply:
it's create url based on your bucket and current time stamp with `overall_export_metadata` suffix.